### PR TITLE
Add basic README

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

File: appengine/standard/endpoints-frameworks-v2/backend/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+1. The backend
+2. The clients
+
+## Running Locally
+
+For more info on running Standard applications locally, see [the getting started documentation](https://cloud.google.com/appengine/docs/python/quickstart).
+
+Install the Endpoints Frameworks v2 library into a vendor directory:
+
+    $ pip install -t lib TODO
```
Comment (jonparrott) [inline appengine/standard/endpoints-frameworks-v2/backend/README.md]: does it make sense to merge this before this is filled out?

**Comment by saicheems** [comment]
WIP... I left a `TODO` placeholder in there for the name of the PyPi lib

PTAL and let me know where I can be more specific/helpful in the instructions.

File: appengine/standard/endpoints-frameworks-v2/backend/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+This sample consists of two parts:
+
+1. The backend
+2. The clients
+
+## Running Locally
+
+For more info on running Standard applications locally, see [the getting started documentation](https://cloud.google.com/appengine/docs/python/quickstart).
+
+Install the Endpoints Frameworks v2 library into a vendor directory:
```
Comment (jonparrott) [inline appengine/standard/endpoints-frameworks-v2/backend/README.md]: We should link to the instructions for setting up vendoring.

File: appengine/standard/endpoints-frameworks-v2/backend/README.md (hunk 1 above)
Comment (saicheems) [inline appengine/standard/endpoints-frameworks-v2/backend/README.md]: Not necessarily, I'm checking with folks to find out what the name will be.
I'm fine with leaving this pull up for a bit, just wanted to start the process with what we've got.

Comment (jonparrott) [inline appengine/standard/endpoints-frameworks-v2/backend/README.md]: Let's do that. Just ping me when you're ready. :)

File: appengine/standard/endpoints-frameworks-v2/backend/README.md (hunk 2 above)
Comment (saicheems) [inline appengine/standard/endpoints-frameworks-v2/backend/README.md]: Done (ptal at the wording)

File: appengine/standard/endpoints-frameworks-v2/backend/README.md (hunk 1 above)
Comment (saicheems) [inline appengine/standard/endpoints-frameworks-v2/backend/README.md]: Done

**Pull request merged**